src/plugins/wizard/public/embeddable/wizard_embeddable.tsx

```diff
@@ -3,125 +3,288 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import React from 'react';
+import _ from 'lodash';
```

Comment:
Let's optimize this to import only `isEqual` and `cloneDeep`

Reply:
sure - this was just some straight porting from the visualize embeddable.

Reply:
fixed